srcpkgs/dnsdist/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'dnsdist'
+pkgname=dnsdist
+version=1.6.0
+revision=1
+build_style=gnu-configure
+configure_args="--with-pic --with-gnu-ld --with-libsodium --with-re2
+ --with-net-snmp --with-libcap --with-libssl --enable-dnscrypt
+ --with-libcrypto=${XBPS_CROSS_BASE}/usr --with-service-user=_dnsdist
+ --with-service-group=_dnsdist --sbindir=/usr/bin --with-gnutls --with-lmdb
+ --enable-dns-over-tls --enable-dns-over-https --sysconfdir=/etc/dnsdist"
+conf_files="/etc/dnsdist/dnsdist.conf"
+hostmakedepends="autoconf automake pkgconf"
+makedepends="libsodium-devel boost-devel re2-devel net-snmp-devel libcap-devel
+ lua54-devel openssl-devel lmdb-devel fstrm-devel libedit-devel gnutls-devel
+ h2o-devel"
+depends="libsodium re2 libnet-snmp libcap lua54 openssl lmdb fstrm libedit
+ gnutls"
+short_desc="Dynamic DNS loadbalancer"
+maintainer="JailBird <jailbird@fdf.net>"
+license="GPL-2.0-only"
+homepage="https://dnsdist.org/"
+distfiles="https://downloads.powerdns.com/releases/${pkgname}-${version}.tar.bz2"
+checksum=a7783a04d8d4ad2b0168ffaaf85ef95d5f557057b0462280684dd799d0cdd292
+
+system_accounts="_dnsdist"
+
+pre_configure() {
+	export CPPFLAGS="-I${XBPS_CROSS_BASE}/usr/include"
+	export LDFLAGS="-L${XBPS_CROSS_BASE}/usr/lib"
+	export PKG_CONFIG_PATH=${XBPS_CROSS_BASE}/usr/lib/pkgconfig
+	export LMDB_LIBS="-llmdb"
+}
+
+post_install() {
+	vsconf dnsdistconf.lua dnsdist.conf
+	vinstall ${FILESDIR}/dnsdist.conf 644 etc/dnsdist
```

Comment:
This should probably use the `dnsdist.conf` upstream provides to install under `/etc`? Why did you go with a custom file?